OracleGoldenGate/21c/README.md

```diff
@@ -5,21 +5,24 @@ These instructions apply to building container images for Oracle GoldenGate vers
 
 ## Contents
 
-* [Before You Start](#before-you-start)
-* [Build an Oracle GoldenGate Container Image](#build-an-oracle-goldengate-container-image)
-* [Running Oracle GoldenGate in a Container](#running-oracle-goldengate-in-a-container)
-  * [Administrative Account Password](#administrative-account-password)
-  * [SSL Certificate](#ssl-certificate)
-  * [Running the Administration Client](#running-the-administration-client)
-* [Known Issues](#known-issues)
-* [License](#license)
-* [Copyright](#copyright)
+- [Oracle GoldenGate 21.3 Microservices Edition Container Images](#oracle-goldengate-213-microservices-edition-container-images)
+  - [Contents](#contents)
+  - [Before You Start](#before-you-start)
+  - [Build an Oracle GoldenGate Container Image](#build-an-oracle-goldengate-container-image)
+  - [Running Oracle GoldenGate in a Container](#running-oracle-goldengate-in-a-container)
+    - [Administrative Account Password](#administrative-account-password)
+    - [SSL Certificate](#ssl-certificate)
+    - [Running the Administration Client](#running-the-administration-client)
+  - [Known Issues](#known-issues)
+  - [License](#license)
+  - [Copyright](#copyright)
```

Comment:
Okay, I will revert.

Reply:
I tried to make the corrections as per this lint: https://github.com/markdownlint/markdownlint

It will be helpful if you can please point to the correct `markdown-lint` that should be used for any future changes.